cli/src/helpers/format.ts

```diff
@@ -0,0 +1,33 @@
+import chalk from "chalk";
+import { execa } from "execa";
+import ora from "ora";
+
+import { type PackageManager } from "~/utils/getUserPkgManager.js";
+import { logger } from "~/utils/logger.js";
+
+// Runs format and lint command to ensure created repository is tidy upon creation
+export const formatProject = async ({
+  pkgManager,
+  projectDir,
+  eslint,
+  biome,
+}: {
+  pkgManager: PackageManager;
+  projectDir: string;
+  eslint: boolean;
+  biome: boolean;
+}) => {
+  logger.info("Formatting project...");
+  const spinner = ora("Running format command\n").start();
+
+  if (eslint) {
+    await execa(pkgManager, ["format:write"], {
+      cwd: projectDir,
+    });
+  } else if (biome) {
+    await execa(pkgManager, ["check:unsafe"], {
```

Comment:
![CleanShot 2025-01-25 at 15 33 33@2x](https://github.com/user-attachments/assets/13bf2e70-6d2a-48ab-aa86-5ba37b4cb85b)

hmm still getting a project that doesn't pass lint

Reply:
i think even if we can't have a matrix that includes biome we should atleast have 1 app scaffold (perhaps the variant with as much selected as possible? biome+auth+trpc+tailwind) to try and surface this as best as possible